### PR TITLE
NTT-109 : Initial pages

### PR DESCRIPTION
**Pull request by parkeradam**
Added all initial pages listed on the ticket. 
Ordered according to ticket in linear flow (check answers before new would you like to add new individual).
Country picking implemented.

### CODE REVIEW

File: app/assets/data/countries/des-codes.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  "CC","CD","CF","CG","CH","CI","CK","CL","CM","CN","CO","CR","CS","CU","CV","CW","CX","CY","CZ","DE","DJ","DK","DM",
+  "DO","DZ","EC","EE","EG","EH","ER","ES","ET","FC","FI","FJ","FK","FM","FO","FR","GA","GB","GD","GE","GF","GG","GH",
+  "GI","GL","GM","GN","GP","GQ","GR","GS","GT","GU","GW","GY","HK","HM","HN","HR","HT","HU","ID","IE","IL","IM","IN",
+  "IO","IQ","IR","IS","IT","JE","JM","JO","JP","KE","KG","KH","KI","KM","KN","KP","KR","KW","KY","KZ","LA","LB","LC",
+  "LI","LK","LR","LS","LT","LU","LV","LY","MA","MC","MD","ME","MF","MG","MH","MK","ML","MM","MN","MO","MP","MQ","MR",
+  "MS","MT","MU","MV","MW","MX","MY","MZ","NA","NC","NE","NF","NG","NI","NL","NO","NP","NR","NU","NZ","OM","PA","PE",
+  "PF","PG","PH","PK","PL","PM","PN","PR","PS","PT","PW","PY","QA","RE","RO","RS","RU","RW","SA","SB","SC","SD","SE",
+  "SG","SH","SI","SJ","SK","SL","SM","SN","SO","SR","SS","ST","SV","SX","SY","SZ","TC","TD","TF","TG","TH","TJ","TK",
+  "TL","TM","TN","TO","TP","TR","TT","TV","TW","TZ","UA","UG","UM","US","UY","UZ","VA","VC","VE","VG","VI","VN","VU",
+  "WF","WS","YE","YT","ZA","ZM","ZW"]
```
Comment (ajchilts) [inline app/assets/data/countries/des-codes.json:12]: Please add a new line here...

File: app/assets/js/countries-autocomplete.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+accessibleAutocomplete.enhanceSelectElement({
+    selectElement: document.querySelector('#country-autocomplete')
+})
```
Comment (ajchilts) [inline app/assets/js/countries-autocomplete.js:3]: Please add a new line

File: app/assets/js/accessible-autocomplete.min.js [hunk 3]
```diff
@@ -0,0 +1,2 @@
...
+//# sourceMappingURL=accessible-autocomplete.min.js.map
```
Comment (ajchilts) [inline app/assets/js/accessible-autocomplete.min.js:2]: New line please :-)

File: app/assets/js/countries-autocomplete.js (hunk 2 above)
Comment (ajchilts) [inline app/assets/js/countries-autocomplete.js:3]: Same

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -2,3 +2,7 @@ $govuk-assets-path: "/ntt-other-individuals-frontend/assets/lib/govuk-frontend/g
 
 @import "lib/govuk-frontend/govuk/all";
 @import "language-toggles.scss";
+
+ul#country-autocomplete__listbox {
+  font-family: arial;
+}
```
Comment (ajchilts) [inline app/assets/stylesheets/application.scss:8]: Please check all files and add a new line where required. :-) 👍

File: app/controllers/CheckYourAnswersController.scala [hunk 5]
```diff
@@ -1,36 +1,79 @@
...
         Json.obj("list" -> answers)
       ).map(Ok(_))
   }
+
+  def onSubmit(): Action[AnyContent] = (identify andThen getData andThen requireData).async {
+
+    implicit request =>
+      val answers = UserAnswers(request.internalId)
+      for {
+        _              <- sessionRepository.set(answers)
```
Comment (ajchilts) [inline app/controllers/CheckYourAnswersController.scala:76]: Why all the spaces?

File: app/controllers/IndexController.scala [hunk 6]
```diff
@@ -1,21 +1,54 @@
...
     implicit request =>
 
       renderer.render("index.njk").map(Ok(_))
   }
+
+  def onSubmit: Action[AnyContent] = (identify andThen getData).async {
+    implicit request =>
+      val answers = UserAnswers(request.internalId)
+      for {
+        _              <- sessionRepository.set(answers)
```
Comment (ajchilts) [inline app/controllers/IndexController.scala:51]: Too many spaces

**Comment by matthewharris1** [comment]
back button functionality looks fine by me :)

**Pull request merged**